myocamlbuild.ml

```diff
@@ -941,7 +941,7 @@ dispatch
                         "gettext" 
                         (BaseEnvLight.load
                            ~allow_empty:true
-                           ~filename:(Lazy.force MyOCamlbuildBase.env_filename)
+                           ~filename:(MyOCamlbuildBase.env_filename)
```

Comment:
Ok, that worked. I could remove the change regarding MyOCamlbuildBase.env_filename.